cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/QueryPlanTest.scala

```diff
@@ -295,7 +295,8 @@ class QueryPlanTest extends DocumentingTestBase with SoftReset {
     profileQuery(
       title = "Create Index",
       text =
-        """The `CreateIndex` operator creates an index on a property for all nodes or relationships having a certain label or relationship type.
+        """The `CreateIndex` operator creates an index. This index can either be on a property for all nodes or relationships having a certain label or relationship type.
+          |Or it can be a lookup index for all nodes or relationships having any label or relationship type.
```

Comment:
```suggestion
          |or it can be a lookup index for all nodes or relationships having any label or relationship type.
```
If the sentence becomes too long, you could rephrase it as two sentences:
The `CreateIndex` operator creates an index. This index can be on a property for all nodes or relationships having a certain label or relationship type.
Otherwise it can be a lookup index for all nodes or relationships having any label or relationship type.

But I much favor the longer sentence.

Reply:
I think the sentence might look too long but I don't particularly care, I don't mind long sentences :P